easybuild/easyblocks/m/mesa.py

```diff
@@ -40,42 +42,78 @@ class EB_Mesa(MesonNinja):
     """Custom easyblock for building and installing Mesa."""
 
     def __init__(self, *args, **kwargs):
-        """Constructor for custom Mesa easyblock: figure out which vales to pass to swr-arches configuration option."""
+        """Constructor for custom Mesa easyblock: figure out which values to pass to swr-arches configuration option."""
 
         super(EB_Mesa, self).__init__(*args, **kwargs)
 
+        self.gallium_configopts = []
+
+        # Check user-defined Gallium drivers
+        gallium_drivers = self.get_configopt_value('gallium-drivers')
+
+        if not gallium_drivers:
+            # Add appropriate Gallium drivers for current architecture
+            arch = get_cpu_architecture()
+            arch_gallium_drivers = {
+                'x86_64': ['swrast', 'swr'],
+                'POWER': ['swrast'],
+            }
+            if arch in arch_gallium_drivers:
+                gallium_drivers = arch_gallium_drivers[arch]
+                # Add configopt for additional Gallium drivers
+                self.gallium_configopts.append('-Dgallium-drivers=' + ','.join(gallium_drivers))
```

Comment:
Why use `self.gallium_configopts` and not `self.cfg.update('configopts', '...')`? Seems cleaner to me and you won't need any class variables, as you can use get_configopt_value in sanity check path. Other classes use that too

Reply:
I would still do the actual update of `configopts` in the `configure_step()`. I think that it is just more natural.

Reply:
But with that change you can remove the whole custom configure step which is also something...
But I can live with that.